misk-events/src/main/kotlin/misk/events/SpooledProducer.kt

```diff
@@ -12,6 +12,20 @@ import misk.hibernate.Session
  * to the event stream, and are done so through a [Producer] transaction.
  */
 interface SpooledProducer {
+  /**
+   * Publishes an event to a sharded spool.
```

Comment:
is it a single event or potentially many?

Reply:
Many, I'll update